packages/manager/modules/hub/src/components/billing-summary/template.html

```diff
@@ -0,0 +1,35 @@
+<div class="manager-hub-billing-summary">
+    <h3 data-translate="hub_billing_summary_title"></h3>
+    <oui-select
+        data-model="$ctrl.billingPeriod"
+        data-items="$ctrl.periods"
+        data-on-change="$ctrl.onPeriodChange()"
+        data-match="label"
+    >
+    </oui-select>
+    <span
+        class="manager-hub-billing-summary_bill-total"
+        data-ng-bind-html="$ctrl.bills && $ctrl.language ? $ctrl.getFormattedPrice($ctrl.bills.total) : '…'"
```

Comment:
getFormattedPrice will be called multiple times based on angular lifecycle dirty check. Can you populate it after fetching bills data.